docs/validators/installation/manage-node.md

```diff
@@ -0,0 +1,31 @@
+---
+id: manage-node
+title: Manage a validator node
+---
+
+Your node will start as an observer. In order to make it into a validator, you will need to have 2500 xEGLD tokens. You can reach out to an admin in our [Telegram community](https://t.me/ElrondValidators) who will gladly help.
+
+Follow these steps to manage your validator node.
+
+Let’s begin!
+
+First, you need to create an Elrond wallet. You can create this wallet on either the [devnet](https://devnet-wallet.elrond.com) or [testnet](https://testnet-wallet.elrond.com).

```

Comment:
somehow duplicated to what we have in mainnet/testnet/devnet. what's the approach here? delete the other scripts docs? or ?

Reply:
The aim is to merge what is available in mainnet/testnet/devnet, in order to have just one script.